app/screens/talk-details/talk-details-screen.tsx

```diff
@@ -98,14 +98,22 @@ const TAB_HOLDER: ViewStyle = { flex: 1 }
 const TAB_STYLE: ViewStyle = { paddingVertical: 7, alignItems: "center", justifyContent: "center" }
 const TAB_CONTAINER = { flexDirection: "row" }
 const WHITE_TEXT = { color: "white" }
+const COMMENT_TEXT = { ...WHITE_TEXT, fontSize: 16, marginTop: 4 }
 const FLEX_ONE = { flex: 1 }
 const FLEX_ROW = { flexDirection: "row" }
 const COMMENT_CONTAINER = { paddingVertical: 15, marginBottom: 50 }
-const COMMENT_STYLE = { paddingHorizontal: 20, marginBottom: 20 }
+const COMMENT_STYLE = {
+  paddingBottom: 15,
+  paddingTop: 20,
+  paddingHorizontal: 20,
+  borderTopWidth: 1,
+  borderTopColor: "rgba(255, 255, 255, .1)",
+}
 const CREATED_BY = { color: "white", fontWeight: "600" }
 const CREATED_AT = { color: "rgba(255, 255, 255, .5)", fontSize: 11, marginLeft: 8, marginTop: 3 }
 const INPUT_CONTAINER = { bottom: 0, position: "absolute", left: 0 }
 const MESSAGE_INPUT = { backgroundColor: "white", height: 50, paddingHorizontal: 8 }
+const REPORT = { marginTop: 5, color: "green" }
```

Comment:
Can we make this text a red-ish color (I think `palette.angry` or `palette.crimson` would work well), and just a bit smaller so it's not the same size as the comment
```suggestion
const REPORT = { marginTop: 5, color: palette.angry, fontSize: 11 }
```

Reply:
<img width="425" alt="Screen Shot 2019-06-18 at 12 29 31 PM" src="https://user-images.githubusercontent.com/6894653/59713640-bf437f00-91c4-11e9-966b-ab9744ffaf30.png">